app/src/main/java/org/mozilla/fenix/tabtray/TabTrayController.kt

```diff
@@ -117,6 +117,7 @@ class DefaultTabTrayController(
         }
 
         showUndoSnackbar(snackbarMessage, snapshot)
+        dismissTabTray()
```

Comment:
Do we need this? Aren't we already calling `dismissTabTrayIfNecessary` in the `showUndoSnackbar` fun?

Reply:
`showUndoSnackbarForTab` is the only one that does `dismissTabTrayIfNecessary`. The one for `showUndoSnackbar` does not. I should probably rename the `forTab` one to explicitly call out that it dismisses the tab tray.